website/_docs/tutorial/kylin_datatype.cn.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: docs-cn
+title:  "数据类型"
+categories: tutorial
+permalink: /cn/docs/tutorial/kylin_datatype.html
+---
+
+## 数据类型
+
+| :--------- | :--------- | :--------- | :--------- | :------------------- |
+| ANY        | CHAR       | VARCHAR    | STRING     | BOOLEAN              |
+| BYTE       | BINARY     | INT        | SHORT      | LONG                 |
+| INTEGER    | TINYINT    | SMALLINT   | BIGINT     | INT4 (for test only) |
```

Comment:
What's the "for test only"? If it is not recommended for end user, just remove it here.